lib/install/phases/bootstrap.go

```diff
@@ -202,7 +202,9 @@ func (p *bootstrapExecutor) configureSystemDirectories() error {
 		filepath.Join(stateDir, "site", "packages", "tmp"),
 		filepath.Join(stateDir, "secrets"),
 		filepath.Join(stateDir, "backup"),
+		filepath.Join(stateDir, "monitoring"),
```

Comment:
I think we also need to add it here: https://github.com/gravitational/gravity/blob/version/5.5.x/lib/ops/opsservice/deploy.go#L88. This is where directories get created/updated during an upgrade.